iStore/Categories/CategoriesGrid.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  CategoriesGrid.swift
+//  iStore
+//
+//  Created by Mohamed Sayed on 14.02.24.
+//
+
+import SwiftUI
+
+struct CategoriesGrid: View {
+    var body: some View {
+        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
```

Comment:
What's `START_MENU_TOKEN` and `End_MENU_TOKEN`?

Reply:
It's not appearing in Xcode only on gitHub.
I haven't started in this view yet but I shouldn't have committed it.